include/json/value.h

```diff
@@ -171,6 +171,9 @@ class JSON_API Value {
     CZString(ArrayIndex index);
     CZString(const char* cstr, DuplicationPolicy allocate);
     CZString(const CZString& other);
+#ifdef JSON_HAS_RVALUE_REFERENCES
+	CZString(CZString&& other);
+#endif
     ~CZString();
     CZString& operator=(CZString other);
```

Comment:
I'm not clear on how adding an overload effects binary compatibility. Since Json.cpp is a static library a version that has this code will call the move operator while one which doesn't will use the old copy version. Since having a (non-virtual) overload doesn't effect the size or layout of instance objects how is this a binary compatibility concern?